examples/cpp/custom_payload_pool/SubscriberApp.hpp

```diff
@@ -17,8 +17,8 @@
  *
  */
 
-#ifndef _FASTDDS_CUSTOM_PAYLOAD_POOL_SUBSCRIBER_HPP_
-#define _FASTDDS_CUSTOM_PAYLOAD_POOL_SUBSCRIBER_HPP_
+#ifndef FASTDDS_EXAMPLES_CPP_CUSTOM_PAYLOAD_POOL__SUBSCRIBER_APP_HPP
```

Comment:
Pls apply for the rest of the file
```suggestion
#ifndef FASTDDS_EXAMPLES_CPP_CUSTOM_PAYLOAD_POOL__SUBSCRIBERAPP_HPP
```